lit_nlp/examples/gcp/model_server.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+r"""A model server for serving models on GCP via Gunicorn."""
+
+from collections.abc import Sequence
+import functools
+import os
+from typing import Optional
+
+from absl import app
+from lit_nlp import dev_server
+from lit_nlp.examples.gcp import constants as lit_gcp_constants
+from lit_nlp.examples.prompt_debugging import models as pd_models
+from lit_nlp.examples.prompt_debugging import utils as pd_utils
+from lit_nlp.lib import serialize
+from lit_nlp.lib import wsgi_app
+
+DEFAULT_DL_FRAMEWORK = 'kerasnlp'
```

Comment:
nit: if possible, all the default values (except `DEFAULT_MODELS `) can be imported from `pd_models`, but I guess this branch is not merged into dev yet.